sdk/include/ndk/i386/asm.h

```diff
@@ -471,6 +471,17 @@ Header Name:
 #endif
 #define EFLAGS_USER_SANITIZE                    0x3F4DD7
 
+
+//
+// HYPERVISOR 
+//
+#define HV_MMU_USE_HYPERCALL_FOR_ADDRESS_SWITCH 0x01
```

Comment:
You're in the middle of CPU-defined flags here. I think it would make more sense elsewhere, e.g. above the CPU feature flags starting in line 640/659